src/date-range-input/date-range-input.tsx

```diff
@@ -526,6 +528,7 @@ export const DateRangeInput = ({
         return (
             <Container
```

Comment:
shouldnt it have the same `onBlur` / `onFocus` / `$focused` updates like how you did for `DateInput`?

Reply:
apologies to expand on this, what about all the consumers of the `InputWrapper`? should they follow the same behaviour? this includes
- `InputGroup`
- `TimePicker`
- `TimeRangePicker`

Reply:
yup, blur/focus handling needs to be reworked across these components. however these can be handled separately, as I'm targeting to migrate them to also use floating-ui and standardise the behaviour